src/test/resources/test-cql/convert-1.cql

```diff
@@ -0,0 +1,303 @@
+library UseofOpioidsfromMultipleProviders_FHIR4 version '1.0.000' 
+
+using FHIR version '4.0.0'
+include FHIRHelpers version '4.0.0' called FHIRHelpers
```

Comment:
Liquibase script inserts libraries with different names and versions. 
It's  FHIRHelpers-4-0-0   ver 0.0.0. Should it look accordingly in  CQL files?

Reply:
Its microservices side stuff. Not this code. They have a different way for doing it for different files. It is also all based on Brens stuff..

Reply:
We should probably discuss that at after-party..
It will probably cause validation issues..

Reply:
It's not an issue, already tracked this down and designed it accordingly. It is how Bren does it in his libs.

Reply:
Hm.. ok. I will talk to you offline.